src/Faker/Core/Color.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Core;
+
+use Faker\Extension;
+
+/**
+ * @experimental This class is experimental and does not fall under our BC promise
+ */
+final class Color implements Extension\ColorExtension
+{
+    protected $allColorNames = [
```

Comment:
```suggestion
    private $allColorNames = [
```

Reply:
Since the class is `final`, what do you think about renaming the field to `$colorNames`?

After all, its value is an array of color names, but probably not of _all_ color names.

Reply:
Yes, definitely! I also just changed the visibility to `private`.